requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.16
 tensorflow>=1.13
+torch>=1.1
```

Comment:
We prefer to not force the user to install Pytorch if they don't need it. Can you put this line in the `requirements_travis.txt` file instead?

Reply:
makes a lot of sense